src/colmap/estimators/bundle_adjustment_test.cc

```diff
@@ -713,7 +711,9 @@ TEST(RigBundleAdjuster, FourView) {
   std::vector<CameraRig> camera_rigs;
   camera_rigs.emplace_back();
   camera_rigs[0].AddCamera(0, Rigid3d());
-  camera_rigs[0].AddCamera(1, Rigid3d());
+  camera_rigs[0].AddCamera(1,
```

Comment:
This was a bug in the test setup before leading to points behind the camera, which now lead to a solver failure. This is fixed now.